### PR TITLE
feat(ui): Add a resize button to the image content card

### PR DESCRIPTION
**Pull request by luckramos**
Add a resize button that opens a modal and let the user change both the width and height of the image

This PR references the issue #90

### CODE REVIEW

File: ui/src/actions/handleResizeImage.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+      setTimeout(() => {
+        location.reload();
+      }, 1500)
+    }
+  }).catch((err) => {
+    toast.dismiss();
+    toast.error("Error: " + err.response.data);
+  })
+}
```
Comment (Caixetadev) [inline ui/src/actions/handleResizeImage.ts:36]: toast.loading returns an id, you can take this id and pass it to success and error so you don't need to use toast.dismiss
```suggestion
  const toastId = toast.loading("Renaming file...");
  axios.put(`/api/cdn/resize/image`, JSON.stringify(data)).then((res) => {
    if (res.status === 200) {
      toast.success("File resized!", { id: toastId, duration: 4000 });
      
      setTimeout(() => {
        location.reload();
      }, 1500)
    }
  }).catch((err) => {
    toast.error("Error: " + err.response.data, { id: toastId, duration: 4000 });
  })
}
```

**Review by Caixetadev — CHANGES_REQUESTED** [review]

**Comment by luckramos** [comment]
fixed in commit [04c9f3d](https://github.com/kevinanielsen/go-fast-cdn/pull/142/commits/04c9f3ddc3255e7117808f08cb351c90ad1a5409)

**Comment by Caixetadev** [comment]
@kevinanielsen LGTM, what do you think? Can we merge?

**Comment by kevinanielsen** [comment]
> @kevinanielsen LGTM, what do you think? Can we merge?

Yeah, looks good - Nice job on this :)

**Pull request merged**